ckan/lib/helpers.py

```diff
@@ -68,6 +68,9 @@
 LEGACY_ROUTE_NAMES = {
     'home': 'home.index',
     'about': 'home.about',
+    'search': 'dataset.search',
+    'organizations_index': 'organization.index',
+    'group_index': 'group.index'
```

Comment:
These two should be supported out of the box (https://github.com/ckan/ckan/blob/master/ckan/lib/helpers.py#L354). It's best to keep this list with only the really necessary legacy routes.

Reply:
@amercader i guess `organizations` will be problem as the blueprint name is `organization`

Reply:
You are absolutely right. This one should go in, I read too fast.